flexget/plugins/output/prowl.py

```diff
@@ -65,9 +67,15 @@ def on_task_output(self, task, config):
                 description = entry['title']
                 log.error('Error rendering jinja description: %s' % e)
 
+            # If url has jinja template, render it
+            try:
+                message_url = entry.render(message_url)
+            except RenderError as e:
+                log.error('Error rendering jinja url: %s' % e)
```

Comment:
You need to supply a default for `message_url` or it'll crash when you try to send if a RenderError as occured.